<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlFIOForm</class>
 <widget class="QWidget" name="ControlFIOForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="labelMulResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MulInput</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReceiveEIO">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive EIO</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMulInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>16777215</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveSetting">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save setting</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFrameCycle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay Port A </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDelayFrameTriggerA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>16777215</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Unit1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Unit2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pulse</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDelayFrameTriggerB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>16777215</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay Port B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSendFrameTrigger">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Unit3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>us</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonFlapOK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Flap OK</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonFlapNG">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Flap NG</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDropLength">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Flap drop count</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Unit3_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Line trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Flap drop ON pulse</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDropFlapONLength">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>16777215</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Unit3_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pulse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame trigger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1 frame cycle</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonTopNG">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Top NG</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonTopOK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Top OK</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonBottomNG">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom NG</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonBottomOK">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom OK</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn0">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn6">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn9">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn10">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn8">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn11">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn13">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN13</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn14">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN14</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn12">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN12</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxIn15">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>IN15</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor pulse length</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMotorPulseCounter">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>XXXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxTriggerDelayParameter">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
   <property name="value">
    <double>200000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger delay parameter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
